.konflux/scripts/render_snapshot.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import time
+
+
+def load_image_refs():
+    return json.loads(os.getenv("IMAGE_REFS", '[]'))
+
+
+def process_component(component, name_suffix):
+    if name_suffix != "":
+        name = f"{component['component']}-{name_suffix}"
+    else:
+        name = component["component"]
+    return {
+        "containerImage": component["ref"],
+        "name": name,
+        "source": {
+            "git": {
+                "revision": component["revision"],
+                "url": component["repository"]
+            }
+        }
+    }
+
+
+def construct_snapshot(
+    snapshot_name_prefix,
+    pipeline_run_name,
+    application,
+    components
+):
+    snapshot_name = f"{snapshot_name_prefix}-{int(time.time())}"
```

Comment:
1. While epoch time is ok-ish, it's not human readable and it can be hard to compare which of two snapshots is more recent since the numbers are big. I suggest using iso8601-ish date here, e.g. `20241218T193201Z` or `20241218_193201z`. We don't need to parse that programmatically so can be flexible but more human-friendly than with the epoch time.
2. WDYT about including the result of the `determine-main-image-tag` task in the snapshot name prefix (i.e. passing that in the pipeline)? I think it will be handy at the time of release to see that yes this is Snapshot of the tagged build (of course, tags recognition by the task is subject of another ticket - ROX-26026).

Reply:
Done in https://github.com/stackrox/stackrox/pull/13577/commits/aa031ef6d3e4cedc0d2ab3e8016caa113d3de7ef